planning/obstacle_cruise_planner/config/obstacle_cruise_planner.param.yaml

```diff
@@ -10,7 +10,6 @@
       min_ego_accel_for_rss: -1.0 # ego's acceleration to calculate RSS distance [m/ss]
```

Comment:
Please apply the same modification to tier4_planning_launch in autoware.universe

Reply:
done